spec/index.bs

```diff
@@ -7156,6 +7157,13 @@ The following enums are supported if and only if the {{GPUFeatureName/"pipeline-
 </dl>
 
 
+## <dfn dfn-type=enum-value dfn-for=GPUFeatureName>shader-float16</dfn> ## {#shader-float16}
+
+Allows 16bit float arithmetic feature in shader and 16bit storage features for GPUBuffer.
```

Comment:
what are the "16bit storage features of GPUBuffer"?

Reply:
I think it means accessing buffer memory through 16bit types in WGSL. (but it needs to be reworded to say that)

Reply:
In principle this is a fine idea.

As noted, there's a lot of detail to be determined.  Support in the field is very uneven, and Vulkan slices the functionality into lots of pieces, e.g.
- some devices can compute in fp16, but not load or store individual fp16 values.
- some devices can load or store individual fp16 values but not directly do fp16 arithmetic.
- ability to load/store fp16 values differs by storage class.  storage buffer, uniform buffer, input/output, workgroup storage.

- Also, you'd need to specify all the places f16 can be used, and behaviour of arithmetic, and error bounds on builtins (if they apply)